packages/mongo-nav/README.md

```diff
@@ -204,3 +160,71 @@ export interface URLSInterface {
   };
 }
 ```
+
+##### DataInterface
+
+```typescript
+export interface AccountInterface {
```

Comment:
`export` doesn't seem meaningful in this context

Reply:
👍